Dockerfile

```diff
@@ -1,22 +1,51 @@
-FROM node:21-slim as base
+FROM node:21-alpine as base
 
-EXPOSE 3000/tcp
 WORKDIR /usr/app
-COPY ./ ./
+COPY ./package.json \
+     ./package-lock.json \
+     ./next.config.js \
+     ./next-env.d.ts \
```

Comment:
This file does not exist as far as I can tell? So my test build of your branch is failing
Maybe I'm missing something

Reply:
I tried creating an empty file with the name just to get further, and it seems like it's gitignored, so I guess this is some sort of local file that I don't have.

Reply:
Good catch, that seems like a generated file. I'll remove that line 👍